### ISSUE TITLE
nnx.Conv does not work as expected

### ISSUE THREAD

**Issue by leson207** [google/flax#4214]
### System information
- Colab
- Latest version (!pip install git+https://github.com/google/flax.git)

### Problem you have encountered:

X = array shape (1,128,288)

conv = nnx.Conv(in_features=conv_dim, out_features=conv_dim, kernel_size = args.kernel_size,
                             padding='CAUSAL', feature_group_count=conv_dim, use_bias=args.conv_bias, rngs=rngs)

output = conv(x)

output shape is not the same as expected
### What you expected to happen:
output' shape = (1,128,128) instead of (1,128,288)

**Comment by cgarciae** [comment]
Hey, currently
```python
assert jax.nn.swish is nnx.swish
```
so I'm not sure what the issue could be here.

**Comment by leson207** [comment]
> Hey, currently
> 
> ```python
> assert jax.nn.swish is nnx.swish
> ```
> 
> so I'm not sure what the issue could be here.

I know that them the same, i mean when i use these functions, which build on each other or just other name(silu,swish), it change my input shape
